VideoWKWebView/AppDelegate.swift

```diff
@@ -31,6 +32,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func applicationDidEnterBackground(_ application: UIApplication) {
+        // Make it works with iOS 13.
+        do {
+            try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback)
+            try AVAudioSession.sharedInstance().setActive(true)
+        } catch {
+            print(error)
+        }
```

Comment:
It might be better to put this code before the player begins, like for instance, inside `videDidLoad` of `Player.swift`.